site/src/atomics/spacing.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+- `<css-property-name>-<size>-<screensize>`. For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md)
+
+### Accepted property names
```

Comment:
I think this page is perhaps suffering from heading overload. Could the following be just plain text?

```
### Accepted property names
### Accepted sizes (🌟 also, should this be `values`?
### Accepted screensizes


### Adding uniform spacing
### Adding spacing to a side
### Adding spacing on a particular screen size
```


```

Reply:
Updated